R/dataset.R

```diff
@@ -137,7 +137,24 @@ Dataset.data <- function(id, filters, row.names = NULL, env = solvebio:::.solveE
 
     tryCatch({
         res <- .request('POST', path=path, body=body, env=env)
-        return(formatSolveBioQueryResponse(res, row.names=row.names))
+        res <- formatSolveBioQueryResponse(res)
+        if (!is.null(col.names)) {
```

Comment:
In the case of dataset queries, since we do the name/title mapping after the results are retrieved, how about adding a flag to the function to turn toggle the feature, lets disable it by default for backwards compatibility: `Dataset.query(use_field_titles=FALSE)` and instead of mapping this on each call of `Dataset.data()`, do it here instead at the end just one time https://github.com/solvebio/solvebio-r/blob/63e36b6a8be7b75b81e94d4f751cd13eccd871f9/R/dataset.R#L213